app/src/main.c

```diff
@@ -11,7 +11,7 @@
 #include <zephyr/bluetooth/audio/bap.h>
 #include <zephyr/bluetooth/audio/bap_lc3_preset.h>
 
-BUILD_ASSERT(strlen(CONFIG_BROADCAST_CODE) <= BT_AUDIO_BROADCAST_CODE_SIZE,
+BUILD_ASSERT(strlen(CONFIG_BROADCAST_CODE) <= BT_ISO_BROADCAST_CODE_SIZE,
```

Comment:
if this is new in upstream zephyr, we should either change it back to the zephyr listed in the project's west.yml .. or bump the same.

Reply:
Saw some issues with the new zephyr version, so lets stick with the old version for now.